tests/molecules/C4_ae/CMakeLists.txt

```diff
@@ -0,0 +1,150 @@
+
+IF (NOT QMC_CUDA)
+ IF (NOT QMC_COMPLEX)
+
+# E_scf_QP=-150.95914112726956
+#  Reference from VMC runs
+  LIST(APPEND C4_SD_AE_QP_SHORT "totenergy" "-150.960566064824 0.074761412109")
+  LIST(APPEND C4_SD_AE_QP_SHORT "variance" "34.741879231108 4.833758192799")
+  LIST(APPEND C4_SD_AE_QP_SHORT "kinetic" "149.744668494694 2.753622966294")
+  LIST(APPEND C4_SD_AE_QP_SHORT "potential" "-300.705234559488 2.761778303546")
+  LIST(APPEND C4_SD_AE_QP_SHORT "eeenergy" "106.589716452578 0.093871118179")
+  LIST(APPEND C4_SD_AE_QP_SHORT "ionion" "56.960598986000 0.001000000000")
+
+  QMC_RUN_AND_CHECK(short-C4_sd-ae-qp-vmc
+                    "${CMAKE_SOURCE_DIR}/tests/molecules/C4_ae"
+                    qmc_short-SD-C4_AE_Mol_QP
+                    qmc_short-SD-C4_AE_Mol_QP.in.xml
+                    1 16
+                    TRUE
+                    0 C4_SD_AE_QP_SHORT # VMC
+                    )
+
+
+
+  LIST(APPEND C4_SD_AE_QP_LONG "totenergy" "-150.960566064824 0.007790118606")
+  LIST(APPEND C4_SD_AE_QP_LONG "variance" "34.741879231108 0.990741221699")
+  LIST(APPEND C4_SD_AE_QP_LONG "kinetic" "149.744668494694 0.331776551096")
+  LIST(APPEND C4_SD_AE_QP_LONG "potential" "-300.705234559488 0.330983245767")
+  LIST(APPEND C4_SD_AE_QP_LONG "eeenergy" "106.589716452578 0.011179248515")
+  LIST(APPEND C4_SD_AE_QP_LONG "ionion" "56.960598986000 0.001000000000")
+
+  QMC_RUN_AND_CHECK(long-C4_sd-ae-qp-vmc
+                    "${CMAKE_SOURCE_DIR}/tests/molecules/C4_ae"
+                    qmc_long-SD-C4_AE_Mol_QP
+                    qmc_long-SD-C4_AE_Mol_QP.in.xml
+                    1 16
+                    TRUE
+                    0 C4_SD_AE_QP_LONG # VMC
+                    )
+
+#Nb_Det = 211
+#Ground:
+#E               =   -151.05936895837109
+#Vnn             =    56.960598986007938
+#Ven             =   -464.01242694605406
+#Vee             =    106.35022537160214
+#T               =    149.64423363007290
+
+
+#Excited:
+#E               =   -150.89066382584013
+#Vnn             =    56.960598986007938
+#Ven             =   -463.18948426388619
+#Vee             =    106.03958987195335
+#T               =    149.30063158008477
+
+
+  LIST(APPEND C4_MD_AE_EXCITED_QP_SHORT "totenergy" "-150.892309364193 0.062212972577")
```

Comment:
Should use E_excited above.

Reply:
Amend or add explanatory comment in the text.

Reply:
Done